websites/docs.temporal.io/src/theme/EditThisPage/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+export default function EditThisPageWrapper() {
+  const feedbackStyle = {
+    marginTop: "20px",
+    padding: "10px 15px",
+
+    borderRadius: "8px",
+    borderLeft: "4px solid #007bff", // A blue accent to match the button
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "flex-start",
+  };
+
+  return (
+    <>
+      <div style={feedbackStyle}>
+        <p style={{ margin: 0, fontWeight: "500" }}>Have feedback on our documentation?</p>
+        <a href="http://t.mp/slack" target="_blank" rel="noopener noreferrer">
+          Join our community Slack channel
+        </a>
```

Comment:
```suggestion
        Post in the #documentation channel in our
        <a href="http://t.mp/slack" target="_blank" rel="noopener noreferrer">
          community Slack
        </a>
```

unclear which channel